packages/clay-card/src/Card.tsx

```diff
@@ -82,7 +85,9 @@ const ClayCard: React.FunctionComponent<IProps> & {
 				'template-card-horizontal': horizontal && interactive,
 				'user-card': isCardType.user,
 			})}
+			href={interactive ? href : undefined}
```

Comment:
I'm not sure about this, why not just leave `href`?

Reply:
Because we default `href = 'javascript:;'` in the destructuring of the props. So we don't want to apply any href if the element is not an anchor. This helps align with the line `const TagHeaderName = interactive ? 'a' : 'div';` as well

Reply:
Hmm, ok makes sense.